HashCalc/HashCalc.cs

```diff
@@ -0,0 +1,53 @@
+using GmodNET.API;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace NativeMath
+{
+	public class HashCalc : IModule
+	{
+		public string ModuleName => "HashCalc";
+
+		public string ModuleVersion => "0.0.1";
+
+		private List<GCHandle> handles;
+
+		private static int CalculateHash(ILua lua)
+		{
+			string hashName = lua.GetString(1);
+			string data = lua.GetString(2);
```

Comment:
I think we have to check that function parameters are valid i.e. we have at lease two params passed, both parameters are strings (or can be automatically converted to strings), the hash algorithm name is valid, etc.